tsdat/transform/converters.py

```diff
@@ -331,6 +331,9 @@ class NearestNeighbor(_ADIBaseTransformer):
     transformation_type: str = "TRANS_SUBSAMPLE"
 
 
+# TODO: Looks like a documentation reference here to `converters.py`. Should be changed
+#  given the refactor.
+
 # tsdat/
```

Comment:
Actually, this is leftover design work that somehow made it in here. This was when we were sketching out the layout for the transform submodules. Old stuff. This can all be deleted

Reply:
I'll remove this when I make a pass through the transform folder.